apps/frontend/src/components/user/PeopleTable.tsx

```diff
@@ -367,12 +368,33 @@ const PeopleTable = ({
             userRole: query.userRole,
           });
 
+          const paginatedFilteredData = data
```

Comment:
I am wondering isn't it better to do the backend filter and search instead of frontend one or maybe I am not getting the point of this so well.

Reply:
Sorry didn't think of this at the time, looking into this I would need to paginate currently 3 places where the data field is used in 3 different places: 

- https://github.com/UserOfficeProject/user-office-core/blob/1075-fix-fap-review-page/apps/frontend/src/components/questionary/questionaryComponents/ProposalBasis/QuestionaryComponentProposalBasis.tsx#L44
- https://github.com/UserOfficeProject/user-office-core/blob/1075-fix-fap-review-page/apps/frontend/src/components/proposalBooking/CreateUpdateVisit.tsx#L88
- https://github.com/UserOfficeProject/user-office-core/blob/1075-fix-fap-review-page/apps/frontend/src/components/fap/Proposals/AssignFapMemberToProposalModal.tsx#L71

As far as I can tell, the first 2 look like they tricky to paginate, as the data that is passed to the PeopleTable is used elsewhere in the file. Am I understanding it correctly

Reply:
Maybe it is just good to open a ticket about it for now and then we can take it from there.

Reply:
Opened https://github.com/UserOfficeProject/issue-tracker/issues/1141